ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -46,25 +46,41 @@
     <:title>{{t 'form.type.label'}}</:title>
     <:body>
       {{#if @model.isNew}}
-        <form.radioGroup
-          @name='types'
-          @selectedValue={{@model.type}}
-          @changed={{@changeType}}
-          as |radioGroup|
-        >
-          <radioGroup.card
-            @label={{t 'resources.target.types.tcp'}}
-            @helperText={{t 'resources.target.help.tcp'}}
-            @value='tcp'
-            @layout='wide'
-          />
-          <radioGroup.card
-            @label={{t 'resources.target.types.ssh'}}
-            @helperText={{t 'resources.target.help.ssh'}}
-            @value='ssh'
-            @layout='wide'
-          />
-        </form.radioGroup>
+        {{#if (feature-flag 'ssh-target')}}
```

Comment:
Can this be simplified to wrap only the SSH card?

Reply:
+1 to apply the conditional Only to the SSH card.

Reply:
done!